package.json

```diff
@@ -23,9 +23,7 @@
     "moment": "^2.10.6",
     "numeral": "^1.5.3",
     "react-addons-shallow-compare": "^0.14.7",
-    "recompose": "^0.15.1"
-  },
-  "peerDependencies": {
+    "recompose": "^0.15.1",
     "react": "^0.14.2",
```

Comment:
@thehindenburg if you want to work on any additional changes this week before the new branch gets released:

* check w/ me first so we don't step on each other's toes
* branch off of the `0.2.1-combined` branch and PR back into that branch.

Thanks!